### PR TITLE
NP-43 : fixed keyword highlighting in course description

### PR DESCRIPTION
**Pull request by enkyuan**
## What Changed

Modified CourseInfoHoverCard.tsx to display description with course and corresponding prerequisite bolded.

### CODE REVIEW

**Review by akevinge — CHANGES_REQUESTED** [review]
The format/lint check is failing. You can run the format via `npm run format`. Also make sure you have the Prettier configured in VSCode.

**Comment by enkyuan** [comment]
@akevinge format/lint error fixed. Should be ready to merge now unless there's something else I'm missing. Please let me know.

File: src/components/planner/CourseInfoHoverCard.tsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ export const CourseInfoHoverCard: FC<CourseInfoHoverCardProps> = ({
         className="z-[999] w-[300px] animate-[slideUpAndFade_0.3s] rounded-md border border-neutral-200 bg-generic-white p-6 shadow-sm"
         sideOffset={5}
       >
+        Hdi
```
Comment (akevinge) [inline src/components/planner/CourseInfoHoverCard.tsx:35]: I think you might have accidently left this in here.

**Review by akevinge — CHANGES_REQUESTED** [review]

File: src/components/planner/CourseInfoHoverCard.tsx (hunk 1 above)
Comment (enkyuan) [reply src/components/planner/CourseInfoHoverCard.tsx:35]: Will do. Just remove it?

Comment (akevinge) [reply src/components/planner/CourseInfoHoverCard.tsx:35]: Yes.

**Review by akevinge — APPROVED** [review]
LGTM

**Pull request merged**